migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/sql_schema_differ_flavour/postgres.rs

```diff
@@ -290,7 +290,7 @@ impl SqlSchemaDifferFlavour for PostgresFlavour {
     }
 }
 
-fn cockroach_column_type_change(columns: Pair<TableColumnWalker<'_>>) -> Option<ColumnTypeChange> {
+fn cockroach_column_type_change(columns: MigrationPair<TableColumnWalker<'_>>) -> Option<ColumnTypeChange> {
```

Comment:
Not a big fan of the `MigrationPair` name, but I understand why. We'll change that if we work on that code again and get frustrated.